src/main/java/com/iridium/iridiumenchants/effects/Heal.java

```diff
@@ -15,9 +15,9 @@ public void apply(LivingEntity player, LivingEntity target, String[] args, ItemS
             amount = 1;
         }
         if (args.length == 3 && args[2].equalsIgnoreCase("target")) {
-            target.setHealth(target.getHealth() + amount);
+            target.setHealth((target.getHealth() +amount > 20) ? 20 : target.getHealth() + amount);
```

Comment:
please use [getMaxHealth](https://jd.bukkit.org/org/bukkit/entity/Damageable.html#getMaxHealth())() instead of hardcoding 20, some plugin change the maxHealth of players.